### DISCUSSION TITLE
With sensor mode , docker "tpotinit" didn't run

### DISCUSSION THREAD

**Discussion by elcazator** [telekom-security/tpotce#1724] [Q&A]
i used install.sh to install the t-pot.when finished ,the docker didn't run.Then i try command "docker compose up",the command throw an error 
![image](https://github.com/user-attachments/assets/df2e8510-7682-4055-a8e4-383ff66da020)
how to fix this Error?

**Comment by statisticalcomponent** [comment]
Hi, in the error it says "WEB_USER is not set or empty". This value is in the .env file in the same folder as install.sh
You can try to execute genuser.sh and copy paste the base64 string (the one in parenthesis) as value for WEB_USER in .env
